src/views/UserProfileView.vue

```diff
@@ -262,7 +262,7 @@
 
   .profile__header{
     top: 0;
-    z-index: 10;
```

Comment:
is there a solution without z-index?

Reply:
The icons have higher z-index(they are clickable and have popups) than the sticky section. It's a simple and only solution in this case.

Reply:
The Icons shouldn't have a z-index as well, as I said, is there a solution WITHOUT z-index? (cc @kenodressel )

Reply:
<img width="889" alt="Screenshot 2020-03-12 at 13 32 27" src="https://user-images.githubusercontent.com/7085525/76517469-06198900-6466-11ea-9cf2-e7122b50779d.png">

Reply:
I could remove the z-index from the icons but still the z-index: 10; should stay for the header, because the profile section is with relative position(z-index is unset) and below the header => the header will go under it without specified z-index.